test/fixtures/project/client/abc/empty-expected.js

```diff
@@ -0,0 +1,3 @@
uire.modules[t])return t}t||(t="/");if(require._core[e])return e;var n=require.modules.path();e=e.charAt(0)==="/"?n.resolve(t,e.substring(1)):e,e=e.match(/^(?:\.\.?\/)/)?n.resolve(t,e):e;var r=i(e,n.join(e,"index"));if(r)return r;throw new Error("Cannot find module '"+e+"' within '"+t+"'")},require.alias=function(e,t){var n=require.modules.path(),r=null;try{r=require.resolve(e+"/package.json","/")}catch(i){r=require.resolve(e,"/")}var s=n.dirname(r),o=(Object.keys||function(e){var t=[];for(var n in e)t.push(n);return t})(require.modules);for(var u=0;u<o.length;u++){var a=o[u];if(a.slice(0,s.length+1)===s+"/"){var f=a.slice(s.length);require.modules[t+f]=require.modules[s+f]}else a===s&&(require.modules[t]=require.modules[s])}},require.define=function(e,t){var n=require._core[e]?"":require.modules.path().dirname(e),r=function(e){return require(e,n)};r.resolve=function(e){return require.resolve(e,n)},r.modules=require.modules,r.define=require.define;var i={exports:{}};require.modules[e]=function(){return require.modules[e]._cached=i.exports,t.call(i.exports,r,i,i.exports,n,e),require.modules[e]._cached=i.exports,i.exports}},typeof process=="undefined"&&(process={}),process.nextTick||(process.nextTick=function(){var e=[],t=typeof window!="undefined"&&window.postMessage&&window.addEventListener;return t&&window.addEventListener("message",function(t){if(t.source===window&&t.data==="browserify-tick"){t.stopPropagation();if(e.length>0){var n=e.shift();n()}}},!0),function(n){t?(e.push(n),window.postMessage("browserify-tick","*")):setTimeout(n,0)}}()),process.title||(process.title="browser"),process.binding||(process.binding=function(e){if(e==="evals")return require("vm");throw new Error("No such module")}),process.cwd||(process.cwd=function(){return"."}),require.define("path",function(e,t,n){function r(e,t){var n=[];for(var r=0;r<e.length;r++)t(e[r],r,e)&&n.push(e[r]);return n}function i(e,t){var n=0;for(var r=e.length;r>=0;r--){var i=e[r];i=="."?e.splice(r,1):i===".."?(e.splice(r,1),n++):n&&(e.splice(r,1),n--)}if(t)for(;n--;n)e.unshift("..");return e}var s=/^(.+\/(?!$)|\/)?((?:.+?)?(\.[^.]*)?)$/;n.resolve=function(){var e="",t=!1;for(var n=arguments.length;n>=-1&&!t;n--){var s=n>=0?arguments[n]:process.cwd();if(typeof s!="string"||!s)continue;e=s+"/"+e,t=s.charAt(0)==="/"}return e=e.replace(/\/\.\//,"/"),e=i(r(e.split("/"),function(e){return!!e}),!t).join("/"),(t?"/":"")+e||"."},n.normalize=function(e){var t=e.charAt(0)==="/",n=e.slice(-1)==="/";return e=i(r(e.split("/"),function(e){return!!e}),!t).join("/"),!e&&!t&&(e="."),e&&n&&(e+="/"),(t?"/":"")+e},n.join=function(){var e=Array.prototype.slice.call(arguments,0);return n.normalize(r(e,function(e){return e&&typeof e=="string"}).join("/"))},n.dirname=function(e){var t=s.exec(e)[1]||"";return t?t==="/"?t:t.substring(0,t.length-1):"."},n.basename=function(e,t){var n=s.exec(e)[2]||"";return t&&n.substr(-1*t.length)===t&&(n=n.substr(0,n.length-t.length)),n},n.extname=function(e){return s.exec(e)[3]||""}});
```

Comment:
Line is too long.<br>Missing semicolon.<br>Expected a conditional expression and instead saw an assignment.<br>Expected an assignment or function call and instead saw an expression.<br>The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.<br>Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.<br>'r' is already defined.<br>'i' is already defined.